docs/hacking_howto.rst

```diff
@@ -90,6 +90,8 @@ These are required for the minimum installation:
 * libxslt and headers
 * libjpeg and headers
 * zlib and headers
+* libssl library
+* libmysqlclient library
```

Comment:
@willkg I meant to check installing Kitsune in Ubuntu.
Whie I try to install in Ubuntu, though I installed all the Requirments, it needed that 2 Library also.
So I proposed you to check again!